Cargo.toml

```diff
@@ -7,12 +7,15 @@ license = "Apache-2.0/MIT"
 repository = "https://github.com/nikomatsakis/rayon"
 documentation = "https://docs.rs/rayon/"
 
+[workspace]
+members = ["rayon-demo", "rayon-core"]
+
 [dependencies]
+rayon-core = { path = "rayon-core" }
```

Comment:
The path-dep is in effect for local builds, but when you publish to crates.io you have to explicitly say the version of `rayon-core` to depend on.  So you do both:

    rayon-core = { path = "rayon-core", version = "0.1" }